### PR TITLE
Minor changes in sd preparation and sending payload

### PR DESCRIPTION
**Pull request by JeffVi**
**SD Preparation**

Fixed issue with the note in step 4 being considered as step 5
Added a note about replacing files (for the bootloader folder merge)

*see https://jeffvi.github.io/switch-guide/user_guide/emummc/sd_preparation/*

**Sending a payload (emummc)**

Using the TE payload to check if the switch is patched for more consistency with the rest of the guide. Any reason not to do it?
Also added precision about the hekate payload since the question is frequently asked.

**Sending a payload (syscfw)**

Added precision about the hekate payload since the question is frequently asked.

Not much, but a lot of questions in switch assistance were about that these days.
For TE instead of Hekate to check if the device is patched, feel free to close the PR if there is any reason not to do it. I should have asked earlier.

### CODE REVIEW

File: docs/user_guide/emummc/sd_preparation.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Atmosphere has its own bootloader, called fusee (primary). For the purposes of t
     2. Copy *the contents of* the Atmosphere `.zip` file to the root of your SD card
     3. Copy the `bootloader` folder from the Hekate `.zip` file to the root of your SD card
     4. Copy the `bootloader` folder from the `bootlogos.zip` file to the root of your SD card
-       - If you're asked to merge the bootloader folders, do so
+          - If you're asked to merge the bootloader folders and replace files, do so
```
Comment (suchmememanyskill) [inline docs/user_guide/emummc/sd_preparation.md:38]: `*and* replace files`? You should probably make that `or replace the files`. This should be done for the sysmmc guide as well in the pr

File: docs/user_guide/emummc/sending_payload.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ Now that the device is in RCM, we will need to send it a payload. The methods ar
 &nbsp;
 
 !!! danger "If nothing happens after you send the payload"
-    If your console's screen remains black after you've sent Hekate, it's possible your payload was corrupted, or that your console is patched. If your payload injector program shows that 0 bytes were sent, then it is definitely patched, so you'll be unable to proceed with the rest of the guide.
+    If your console's screen remains black after you've sent TegraExplorer, it's possible your payload was corrupted, or that your console is patched. If your payload injector program shows that 0 bytes were sent, then it is definitely patched, so you'll be unable to proceed with the rest of the guide.
```
Comment (suchmememanyskill) [inline docs/user_guide/emummc/sending_payload.md:124]: Probably worth specifying that not specifically only tegraexplorer works for this, but any payload, like `Tegraexplorer (or any other payload)`

**Review by suchmememanyskill — APPROVED** [review]

**Review by el-emmo — APPROVED** [review]

File: docs/user_guide/sysnand/sd_preparation.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ Atmosphere has its own bootloader, called fusee (primary). For the purposes of t
     2. Copy *the contents of* the Atmosphere `.zip` file to the root of your SD card
     3. Copy the `bootloader` folder from the Hekate `.zip` file to the root of your SD card
     4. Copy the `bootloader` folder from the `bootlogos.zip` file to the root of your SD card
-       - If you're asked to merge the bootloader folders, do so
+          - If you're asked to merge the bootloader folders or replace the files, do so
```
Comment (noirscape) [inline docs/user_guide/sysnand/sd_preparation.md:42]: Needs a dot here.

File: docs/user_guide/emummc/sd_preparation.md [hunk 4]
```diff
@@ -35,7 +35,7 @@ Atmosphere has its own bootloader, called fusee (primary). For the purposes of t
     2. Copy *the contents of* the Atmosphere `.zip` file to the root of your SD card
     3. Copy the `bootloader` folder from the Hekate `.zip` file to the root of your SD card
     4. Copy the `bootloader` folder from the `bootlogos.zip` file to the root of your SD card
-       - If you're asked to merge the bootloader folders, do so
+          - If you're asked to merge the bootloader folders or replace the files, do so
```
Comment (noirscape) [inline docs/user_guide/emummc/sd_preparation.md:38]: Needs a dot here.

**Review by noirscape — CHANGES_REQUESTED** [review]
Missing a few punctuation dots, fix those, then its "looks good to me".

**Review by noirscape — APPROVED** [review]

**Pull request merged**